### ISSUE TITLE
clli SEI documentation is missing a sentence

### ISSUE THREAD

**Issue by ledoge** [AOMediaCodec/libavif#1727]
https://github.com/AOMediaCodec/libavif/blob/f647313b92f5399203379194d73525aa56247aad/include/avif/avif.h#L518-L520

Here, the calculation of E_Max is described, but not how the light level is calculated from it. I don't have access to the cited ISO/IEC standard, but this is the next sentence in H.274:

```
The light level corresponding to the stimulus is then defined as the CIE 1931 luminance
corresponding to equal amplitudes of E_Max for all three colour primary intensities for red, green, and blue (with appropriate
scaling to reflect the nominal luminance level associated with peak white – e.g., ordinarily scaling to associate peak white
with 10 000 candelas per square metre when vui_transfer_characteristics is equal to 16).
```

**Comment by maryla-uc** [comment]
Thanks @y-guyon for digging into the specs. For context, I looked at this (and added the quoted comment in avif.h) when I implemented clli computation in #1724, see here https://github.com/AOMediaCodec/libavif/blob/9d4ac50afc8417e6d5d0b13a1417bc6a4253f30a/src/gainmap.c#L250
I found the specs a bit confusing but my interpretation was also that luminance = E_Max.

**Comment by ledoge** [comment]
> If my understanding is correct, we have
> 
>     * Y = 0.17697 × EMax + 0.81240 × EMax + 0.01063 × EMax
> 
> 
> which gives Y = EMax.

Those specific numbers are for the CIE RGB color space, but the XYZ primaries for any RGB color space are usually normalized such that Y_R + Y_G + Y_B = 1, so it works out all the same. The scale factor for calculating the "light level" in cd/m^2 then depends on the normalization of E_Max and the XYZ primaries, e.g. assuming Y_R + Y_G + Y_B = 1 and E_Max is calculated from [0, 1] normalized pixel values, "light level" = Y * 10 000 for the PQ EOTF. Since these assumptions don't necessarily hold, it makes sense for the spec to just refer to it as "appropriate scaling".

The spec is written in a confusing way, but the calculated result makes sense as an EOTF-encoded upper bound on the individual channel values, because a tone mapping curve calculated from this value is guaranteed to not cause clipping on any of the channels.
